dependency/client_set.go

```diff
@@ -521,3 +535,45 @@ func (c *ClientSet) Stop() {
 		c.nomad.httpClient.Transport.(*http.Transport).CloseIdleConnections()
 	}
 }
+
+func prepareK8SServiceTokenAuth(
+	i *CreateVaultClientInput,
+	client *vaultapi.Client,
+) (err error) {
+	opts := make([]vaultkubernetesauth.LoginOption, 0, 2)
+
+	switch {
+	case i.K8SServiceAccountToken != "":
+		opts = append(opts, vaultkubernetesauth.WithServiceAccountToken(
+			i.K8SServiceAccountToken,
+		))
+	case i.K8SServiceAccountTokenPath != "":
+		opts = append(opts, vaultkubernetesauth.WithServiceAccountTokenPath(
+			i.K8SServiceAccountTokenPath,
+		))
+	default:
+		// The Kubernetes service account token JWT will be retrieved
+		// from /var/run/secrets/kubernetes.io/serviceaccount/token.
```

Comment:
Should be /run/secrets/kubernetes.io/serviceaccount/token

Reply:
@yellowmegaman changed